drivers/dynamixel/dynamixel.c

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2017 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     drivers_dynamixel
+ * @{
+ *
+ * @file
+ * @brief       Driver implementation for Dynamixel devices
+ *
+ * @author      Loïc Dauphin <loic.dauphin@inria.fr>
+ *
+ * @}
+ */
+
+#include "dynamixel.h"
+
+#include "dynamixel_protocol.h"
+#include "dynamixel_reader.h"
+#include "dynamixel_writer.h"
+
+#include "periph/uart.h"
+#include "periph/gpio.h"
+#include "xtimer.h"
+
+#include <string.h>
+
+#define ENABLE_DEBUG 1
+#include "debug.h"
+
+static size_t _dynamixel_stream_write(dynamixel_stream_t *stream, const uint8_t *buffer, size_t size)
+{
+    gpio_set(GPIO_PIN(PORT_B, 5));
```

Comment:
This is a proof of concept with an opencm904.
Of course, the user will specify the right gpio pin with the interface ^^